docs/concepts/backup-restore.md

```diff
@@ -0,0 +1,39 @@
+# Backup and restore
+
+Kubernetes uses the Etcd at database to store the resource definitions. Gardener supports the backup and restore of etcd. It's Shoot owners responsibility to backup their workload data.
+
+
+Gardener uses [etcd-backup-restore] component to backup the etcd backing the Shoot cluster regularly and restore in case of disaster. It is deployed as sidecar via [etcd-druid]. This doc mainly focus on the backup and restore configuration used by gardener to while deploying these components. For more details on the design and internal implementation details of it please take a look at the [GEP-06](../06-etcd-druid.md) and documentation on individual repository.
```

Comment:
```suggestion
Gardener uses [etcd-backup-restore] component to backup the etcd backing the Shoot cluster regularly and restore in case of disaster. It is deployed as sidecar via [etcd-druid]. This doc mainly focuses on the backup and restore configuration used by gardener when deploying these components. For more details on the design and internal implementation details, please refer [GEP-06](../06-etcd-druid.md) and documentation on individual repository.
```